cnxauthoring/views.py

```diff
@@ -36,6 +36,10 @@
 from .storage import storage
 from . import utils
 
+NO_CACHE = (0, {'public': True})
+TIMED_CACHE = (datetime.timedelta(
+    weeks=1, days=0, hours=0, minutes=0, seconds=0), {'public': True})
```

Comment:
It isn't. I thought that it would be good to keep in a TIMED_CACHE example just in case it proved useful in the future.